utils/build/webpack.base.js

```diff
@@ -74,11 +74,7 @@ found at http://www.apache.org/licenses/LICENSE-2.0
           {
             loader: '@svgr/webpack',
             options: {
-              svgoConfig: {
-                plugins: {
-                  removeViewBox: false
-                }
-              }
+              externalConfig: path.resolve('../../node_modules/@zendeskgarden/svg-icons/.svgo.yml')
```

Comment:
replace with...

```json
                svgoConfig: {
                  plugins: [
                    { removeViewBox: false },
                    {
                      addAttributesToSVGElement: {
                        attributes: [
                          {
                            focusable: false
                          }
                        ]
                      }
                    }
                  ]
                }
```

The rest of the config has already been handled by `svg-icons` and shouldn't be duplicated here. Also, the SVGR config should be externalized and shared between webpack and styleguidist.

Reply:
I've added a local `.svgo.yml` file with this configuration